### PR TITLE
Remove SimplifyUselessVariableRector from code quality level

### PR DESCRIPTION
**Pull request by Harfusha**

### CODE REVIEW

File: src/Config/Level/CodeQualityLevel.php [hunk 1]
```diff
@@ -42,7 +42,6 @@
 use Rector\CodeQuality\Rector\FuncCall\SimplifyStrposLowerRector;
 use Rector\CodeQuality\Rector\FuncCall\SingleInArrayToCompareRector;
 use Rector\CodeQuality\Rector\FuncCall\UnwrapSprintfOneArgumentRector;
-use Rector\CodeQuality\Rector\FunctionLike\SimplifyUselessVariableRector;
```
Comment (samsonasik) [inline src/Config/Level/CodeQualityLevel.php:45]: please remove the list in line 116 as well

**Comment by Harfusha** [comment]
fixed

File: src/Config/Level/CodeQualityLevel.php (hunk 1 above)
Comment (samsonasik) [reply src/Config/Level/CodeQualityLevel.php:45]: this line should be removed as well on registering rule, not only use statement:

https://github.com/rectorphp/rector-src/blob/ed6b9124d64a80b10e6e81153353b8e59cd1d8e1/src/Config/Level/CodeQualityLevel.php#L117

on your PR, it on line 116 as removed one line

**Review by samsonasik — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thank you @Harfusha :+1:

**Pull request merged**